src/lib/zod/schemas.ts

```diff
@@ -1,6 +1,9 @@
 import { programmes } from "$lib/utils/programmes";
 import { z } from "zod";
 
+export const emptySchema = z.object({}); // for forms without a body
+export type EmptySchema = typeof emptySchema;
```

Comment:
This is very odd to me. Zod is schema validation library. What's the point of validating an empty object?

Reply:
Superforms require a zod schema, so for a form where no form data is expected, I think using an empty schema makes sense